src/index.ml

```diff
@@ -758,7 +758,9 @@ struct
           | `Aborted -> `Aborted)
     in
     match merge_result with
-    | `Aborted -> (`Aborted, Mtime.Span.zero)
+    | `Aborted ->
+        IO.close merge;
```

Comment:
maybe you can also unlink the file in this case?

Reply:
By using `IO.clear` instead of `IO.close`?